### PR TITLE
Removed SDK hotkey and using event hotkey

### PR DESCRIPTION
**Pull request by Noitidart**
This should address issue #5 and allow the hotkey to work from non-`navigator:browser` type windows. So hotkey from Browser Console window and etc should work now. Untested.

### CODE REVIEW

**Comment by djsmentya** [comment]
@Noitidart not working, I have an error

```
➜  StayOnTop git:(patch-2) ✗ jpm run
JPM [info] Starting jpm run on Stay On Top
JPM [info] Creating a new profile
console.error: stayontop:
JPM [error]   Message: SyntaxError: missing ; before statement
  Stack:
    @resource://stayontop-at-codefusion-dot-ca/index.js:24:undefined
run@resource://gre/modules/commonjs/sdk/addon/runner.js:145:19
startup/</<@resource://gre/modules/commonjs/sdk/addon/runner.js:86:7
Handler.prototype.process@resource://gre/modules/Promise-backend.js:922:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise-backend.js:801:7
this.PromiseWalker.scheduleWalkerLoop/<@resource://gre/modules/Promise-backend.js:740:39


console.error: stayontop:
  Message: SyntaxError: missing ; before statement
  Stack:
    @resource://stayontop-at-codefusion-dot-ca/index.js:24:undefined
run@resource://gre/modules/commonjs/sdk/addon/runner.js:145:19
startup/</<@resource://gre/modules/commonjs/sdk/addon/runner.js:86:7
Handler.prototype.process@resource://gre/modules/Promise-backend.js:922:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise-backend.js:801:7
this.PromiseWalker.scheduleWalkerLoop/<@resource://gre/modules/Promise-backend.js:740:39

```

**Comment by Noitidart** [comment]
Thanks @djsmentya for that test. Please try now I had a typo.

File: index.js [hunk 1]
```diff
@@ -55,10 +57,36 @@ exports.main = function (options, callbacks) {
...
+    for (let window of windows.browserWindows) {
+        viewFor(window).addEventListener('keyup', hotkeyPress, false);
+    }
+    
+    windows.on('open', function(window) {
+        viewFor(window).addEventListener('keyup', hotkeyPress, false);
+    });
+}
+
+function hotkeyPress = function(e) {
```
Comment (djsmentya) [inline index.js]: looks not good

**Comment by djsmentya** [comment]
fix other typos locally but it is not work on my mac for inspector window ((

**Comment by Noitidart** [comment]
Haha thanks for those catches. Is it showing any errors in your browser console?

**Comment by djsmentya** [comment]
This log on start
```
➜  StayOnTop git:(patch-2) ✗ jpm run
JPM [info] Starting jpm run on Stay On Top
JPM [info] Creating a new profile
console.error: stayontop:
JPM [error]   Message: TypeError: windows.on is not a function
  Stack:
    initHotkey@resource://gre/modules/commonjs/toolkit/loader.js -> resource://stayontop-at-codefusion-dot-ca/index.js:73:5
exports.main@resource://gre/modules/commonjs/toolkit/loader.js -> resource://stayontop-at-codefusion-dot-ca/index.js:62:5
run@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/addon/runner.js:151:1
startup/</<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/addon/runner.js:86:7
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:922:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:801:7
this.PromiseWalker.scheduleWalkerLoop/<@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:740:39


JPM [error] *************************
A coding exception was thrown in a Promise resolution callback.
See https://developer.mozilla.org/Mozilla/JavaScript_code_modules/Promise.jsm/Promise

Full message: TypeError: windows.on is not a function
Full stack: initHotkey@resource://gre/modules/commonjs/toolkit/loader.js -> resource://stayontop-at-codefusion-dot-ca/index.js:73:5
exports.main@resource://gre/modules/commonjs/toolkit/loader.js -> resource://stayontop-at-codefusion-dot-ca/index.js:62:5
run@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/addon/runner.js:151:1
startup/</<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/addon/runner.js:86:7
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:922:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:801:7
this.PromiseWalker.scheduleWalkerLoop/<@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:740:39

*************************
console.error: stayontop:
  Message: TypeError: windows.on is not a function
  Stack:
    initHotkey@resource://gre/modules/commonjs/toolkit/loader.js -> resource://stayontop-at-codefusion-dot-ca/index.js:73:5
exports.main@resource://gre/modules/commonjs/toolkit/loader.js -> resource://stayontop-at-codefusion-dot-ca/index.js:62:5
run@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/addon/runner.js:151:1
startup/</<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/addon/runner.js:86:7
Handler.prototype.process@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:922:23
this.PromiseWalker.walkerLoop@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:801:7
this.PromiseWalker.scheduleWalkerLoop/<@resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js:740:39
```

this log when i open inspector
```
console.warn: Asynchronous operation was aborted as selection changed.
```
then I click `show inspector in window` and press hot key, but no effect
but for main window with tabs it work as expected.

**Comment by Noitidart** [comment]
Wow that's weird. Can you please try changing the `windows.on('open',` to `windows.browserWindows.on('open',`

**Comment by djsmentya** [comment]
no errors anymore but hotkey not affect inspector

**Comment by Noitidart** [comment]
Dang. Alrighty, I'll look deeper into it and let you know.

**Comment by Noitidart** [comment]
@djsmentya  please try now, it works for me. This time I tested it haha. I tested on Win10

**Comment by djsmentya** [comment]
Yeh 😀 it wokrs! but console logs errors for each pressed button 
```
console.warn: Asynchronous operation was aborted as selection changed.
console.error: stayontop:
  in keypress, e.key:
  `
console.error: stayontop:
  in keypress, e.key:
  Control
console.error: stayontop:
  in keypress, e.key:
  `
console.error: stayontop:
  in keypress, e.key:
  Control
console.error: stayontop:
  in keypress, e.key:
  `
console.error: stayontop:
  in keypress, e.key:
  Control
console.error: stayontop:
  in keypress, e.key:
  d
console.error: stayontop:
  in keypress, e.key:
  a
console.error: stayontop:
  in keypress, e.key:
  a
console.error: stayontop:
  in keypress, e.key:
  s
console.error: stayontop:
```

**Comment by djsmentya** [comment]
Build created and installed to my firefox developer edition! ☺️ 🌟

**Comment by Noitidart** [comment]
Woohoo!! Yeah The console logging should be removed on production, I left it as I waned to make sure it worked for you.

**Pull request merged**